LoopKitUI/Extensions/Keyboard.swift

```diff
@@ -16,10 +16,10 @@ public final class Keyboard: ObservableObject {
         public var animationDuration: TimeInterval = 0.25
     }
 
-    @Published var state = State()
+    @Published public var state = State()
     private var keyboardFrameChangeCancellable: AnyCancellable?
 
-    static let shared = Keyboard()
+    public static let shared = Keyboard()
```

Comment:
Can you use the public `onKeyboardStateChange` modifier from KeyboardAware.swift instead of making these public?

Reply:
Those are remnants from when I was trying to modify the keyboard, so they aren't needed now. Removed